crates/trie/sparse/src/trie.rs

```diff
@@ -294,8 +294,6 @@ pub struct RevealedSparseTrie<P = DefaultBlindedProvider> {
     prefix_set: PrefixSetMut,
     /// Optional tracking of trie updates for later use.
     updates: Option<SparseTrieUpdates>,
-    /// Reusable buffer for RLP encoding of nodes.
```

Comment:
Actually, could we keep the field and just pass the field as an argument?

Reply:
Alright, done